BasePolicies/CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Node.js Community Contributing Guide 1.0
+
+This document describes a very simple process suitable for most projects
+in the Node.js ecosystem. Projects are encouraged to adopt this whether they
+are hosted in the Node.js or not.
+
+The goal of this document is to create a contribution process that:
+
+* Encourages new contributions.
+* Encourages contributors to remain involved.
+* Avoids unnecessary processes and beuracracy whenever possible.
+* Creates a transparent decision making process which makes it clear how
+contributors can be involved in decision making.
+
+This document is based on much prior art in the Node.js community, io.js,
+and the Node.js project.
+
+## Vocabulary
+
+* A **Contributor** is any individual creating or commenting on an issue or pull request.
+* A **Committer** is a subset of contributors who have been given write access to the repository.
+* A **TC** is a group of committers representing the required technical expertise to 
+resolve rare disputes.
+
+# Logging Issues
+
+Log an issue for any question or problem you might have. When in doubt, log an issue, 
+any additional policies about what to include will be provided in the responses.
+
+Committers may direct you to another repository, ask for additional clarifications, and
+add appropriate metadata before the issue addressed.
+
+Please be courteous, respectful, and follow the project's Code of Conduct.
+
+# Contributions
+
+Any change to resources in this repository must be pull requests. This applies to all change
+to documentation, code, binary files, etc. Even long term committers and TC members must be
+pull requests.
+
+No pull request can be merged by the person that created it.
+
+For non-trivial contributions pull requests should sit for at least 36 hours to ensure that
+contributors in other timezones have time to review.
+
+The default for each contribution is that it is accepted once no commiter has an objection.
+There is no additional "sign off" process for contributions to land. Once all issues brought
+by committers is addressed it can be landed by any committer. The majority of contributions
+should land this way without escalation to the TC.
+
+If a contribution is controversial and committers cannot agree about how to get it to land
+or if it should land then it should be escalated to the TC. TC members should regularly
+discuss pending contributions in order to find a resolution.
+
+# Becoming a Committer
+
+All contributors who send a non-trivial contribution should be on-boarded in a timely manner,
```

Comment:
This is a good point. I would like to keep the language simple for the 99% case but would like to find something that is less exclusionary.

> On Feb 2, 2016, at 9:24AM, Jeremiah Senkpiel <notifications@github.com> wrote:
> 
> In BasePolicies/CONTRIBUTING.md <https://github.com/nodejs/TSC/pull/40#discussion_r51602099>:
> 
> > +
> > +For non-trivial contributions pull requests should sit for at least 36 hours to ensure that
> > +contributors in other timezones have time to review.
> > +
> > +The default for each contribution is that it is accepted once no commiter has an objection.
> > +There is no additional "sign off" process for contributions to land. Once all issues brought
> > +by committers is addressed it can be landed by any committer. The majority of contributions
> > +should land this way without escalation to the TC.
> > +
> > +If a contribution is controversial and committers cannot agree about how to get it to land
> > +or if it should land then it should be escalated to the TC. TC members should regularly
> > +discuss pending contributions in order to find a resolution.
> > +
> > +# Becoming a Committer
> > +
> > +All contributors who send a non-trivial contribution should be on-boarded in a timely manner,
> we don't add committers who haven't gotten a change landed.
> 
> This isn't necessarily true. See: Me <https://github.com/nodejs/node/commits/master?author=fishrock123&page=2> 
> We also added domenic within the context of the time primarily due to issue contributions, and I'm pretty sure we were considering at least one other, though I forget who that was.
> 
> I don't think we are (or should be) "exceptions" per-se. If someone wants to contribute in the form of issues, that is good.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/nodejs/TSC/pull/40/files#r51602099>.
>